charts/px-central/templates/px-backup/pxcentral-prometheus.yaml

```diff
@@ -522,17 +522,20 @@ spec:
 
     - alert: BackupAlert
       annotations:
-        description: {{`The Backup "{{ $labels.name 
-          }}" {{- if eq .Labels.schedule_name "" }} 
-          {{ else }} part of backup schedule "{{ $labels.schedule_name}}" 
-          {{ end }} taken for cluster 
-          "{{ $labels.cluster}}" on namespace 
-          "{{ $labels.backup_namespace}}" has failed.`}}
-        summary: Backup failed
-      expr: pxbackup_backup_status == 4
-      for: 1m
-      labels:
-        severity: critical
+        - alert: BackupAlert
```

Comment:
I think these 2 lines have been repeated, please check:
```    
- alert: BackupAlert
      annotations:
```

Reply:
goo catch, Thanks, missed this in cherry pic